addons/html_editor/static/src/utils/dom_info.js

```diff
@@ -225,7 +225,7 @@ export function isVisibleTextNode(testedNode) {
  * @param {Node} node
  * @returns {boolean}
  */
-const selfClosingElementTags = ["BR", "IMG", "INPUT"];
+const selfClosingElementTags = ["BR", "IMG", "INPUT", "T"];
```

Comment:
Although `T` can be a `selfClosingElement`, it could also not e a `selfClosingElement`.
It is self closing only if it has a `t-esc`, `t-raw`, `t-out`, or `t-set` attribute.
Is it safe to always consider it self closing ? Why do you need it?
If it is safe, can you add a comment about it to make sure the reasoning remains nearby?

Reply:
I need it for the system to consider it during delete FWD and BWD, otherwise it just ignore it and delete on more character

Reply:
maybe we have to restric this to only T element with t-esc or t-raw, I don't know.
Probably the all T element edition need a big improvement